packages/dashboard/src/DashboardLayout.tsx

```diff
@@ -140,13 +141,19 @@ export function DashboardLayout({
             CType.WrappedComponent.prototype.isReactComponent != null) ||
           (CType.prototype != null && CType.prototype.isReactComponent != null);
 
+        // For some unknown reason, ReactIs.isForwardRef returns false, but comparing $$typeof directly works
+        const isForwardRef =
+          !isWrappedComponent(CType) && CType.$$typeof === ReactIs.ForwardRef;
```

Comment:
Ahh good catch. I think I misread indent levels when looking at that code.

I'm going to make a util for detecting if we should forward ref because we will be wrapping components in 2 places. Might be able to make a util to just create the wrapped component actually.